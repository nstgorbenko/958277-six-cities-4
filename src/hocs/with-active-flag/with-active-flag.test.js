import React from "react";
import renderer from "react-test-renderer";

import withActiveFlag from "./with-active-flag.js";

const MockComponent = () => <div />;

describe(`withActiveFlag HOC rendering`, () => {
  it(`should render correctly`, () => {
    const MockComponentWrapped = withActiveFlag(MockComponent);

    const tree = renderer
      .create(<MockComponentWrapped
        isActive={false}
        onActiveChange={() => {}}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
