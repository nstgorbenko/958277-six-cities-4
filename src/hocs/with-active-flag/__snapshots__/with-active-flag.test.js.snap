// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withActiveFlag HOC rendering renders correctly 1`] = `<div />`;
